hyperon_das/das.py

```diff
@@ -17,15 +17,18 @@
 
 
 class DistributedAtomSpace:
-    def __init__(self, **kwargs: Optional[Dict[str, Any]]) -> None:
-        atomdb_parameter = kwargs.get('atomdb', 'ram')
-        query_engine_parameter = kwargs.get('query_engine', 'local')
-
-        if atomdb_parameter == "ram":
+    def __init__(self, system_parameters: Dict[str, Any] = {}, **kwargs) -> None:
+        if not system_parameters.get('running_on_server'):
```

Comment:
Please make an explicit private method `_set_default_system_parameters()` to setup default parameters as we'll probably have other cases where we'll want to do the same. I believe the algorithm should be as you did:

```
def __init__(self, system_parameters...):
   ...
   self.system_parameters = system_parameters
   self._set_default_system_parameters()
   ...
```

```
def _set_default_system_parameters():
    if not system_parameters.get('running_on_server'):
        system_parameters['running_on_server'] = False
```

Reply:
An alternative design would be to use a dataclass `SystemParameters` instead of a dict. This way defaults would be put in the dataclass' constructor. This option has the advantage ensuring the use of the right parameter name across the code, which otherwise would be detected only in runtime. I'll let you decide.